tests/isotope_qty_test.py

```diff
@@ -1,7 +1,15 @@
 import datetime
 from dateutil.parser import parse as dateutil_parse
 import copy
-import numpy as np
+
+try:
+    from mpmath import mp as xp
```

Comment:
Is `mpmath` something we would want to include in `requirements.txt`?

Reply:
That would be ideal. It has gotten a little messy trying to account for both numpy and mpmath